site/docs/master/httpproxy.md

```diff
@@ -254,31 +254,48 @@ The TLS **Minimum Protocol Version** a vhost should negotiate can be specified b
 
 #### Upstream TLS
 
-A HTTPProxy can proxy to an upstream TLS connection by first annotating the upstream Kubernetes service with: `projectcontour.io/upstream-protocol.tls: "443,https"`.
+A HTTPProxy can proxy to an upstream TLS connection by first annotating the upstream Kubernetes Service with the `projectcontour.io/upstream-protocol.tls` annotation.
```

Comment:
We should call out here (and probably the annotations sections maybe), that you can alternatively use the `Spec.Routes.Services.Protocol` to specify the same information and not use the annotation.

Reply:
Yep, that's a good idea. Fixed.